stream-transformation-test/stream-transformation-it/src/test/java/uk/gov/justice/framework/tools/replay/TestEventLogJdbcRepository.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.justice.framework.tools.replay;
+
+
+import static uk.gov.justice.services.test.utils.common.reflection.ReflectionUtils.setField;
+
+import uk.gov.justice.services.eventsourcing.repository.jdbc.AnsiSQLEventLogInsertionStrategy;
+import uk.gov.justice.services.eventsourcing.repository.jdbc.event.EventJdbcRepository;
+import uk.gov.justice.services.jdbc.persistence.JdbcRepositoryHelper;
+
+import javax.sql.DataSource;
+
+/**
+ * Standalone repository class to access event streams. To be used in integration testing
+ */
+public class TestEventLogJdbcRepository extends EventJdbcRepository {
```

Comment:
Do these classes not exist in the framework for us just to depend upon? or do we need to duplicate.